README.md

```diff
@@ -15,6 +15,10 @@ This pull request is where the changes are reviewed.
 If you wish to add new divergent changes to this fork of Fuser,
 open a pull request on the Mountpoint repository branching from the `fuser/fork` branch and use `fuser/fork` as the base branch when creating the pull request.
 
+### Fork Pulishing
```

Comment:
The link is for publishing the "client" crates. It's not clear to me whether this crate will follow exactly the same process.

I suggest we keep this section out for now.